Makefile

```diff
@@ -112,6 +114,80 @@ enwiki_tuning_reports: \
 	tuning_reports/enwiki.wp10.md \
 	tuning_reports/enwiki.nettrom_wp10.md
 
+########################## Persian Wikipedia ###################################
+
+# https://quarry.wmflabs.org/query/25457
+datasets/fawiki.sampled_revisions.300.json:
+	wget https://quarry.wmflabs.org/run/244123/output/0/json-lines?download=true -qO- > $@
+
+datasets/fawiki.labelings.300.json:
+	./utility fetch_labels \
+		https://labels.wmflabs.org/campaigns/fawiki/71/ > $@
```

Comment:
Should we also pull in the labels from campaign 70 (the pilot)?

Reply:
Yup, done